RecoPixelVertexing/PixelTriplets/plugins/CAHitQuadrupletGeneratorGPU.cu

```diff
@@ -330,7 +396,7 @@ void CAHitQuadrupletGeneratorGPU::launchKernels(const TrackingRegion &region,
 
   auto nhits = hh.nHits;
   assert(nhits <= PixelGPUConstants::maxNumberOfHits);
-  auto blockSize = 64;
+  auto blockSize = 128;
```

Comment:
This, most likely, is either a mistake of mine or I got this in from other commits.